CHANGELOG.rst

```diff
@@ -4,13 +4,17 @@ History
 Version 2.3.7
 ^^^^^^^^^^^^^
 
+- **This version will be the last release to support Python 2.7.**
+
 - Fix bug in samefile on Windows when file does not exist.
 
 - Add newline parameter for write_text (see issue #64).
 
-- Project migrated to jazzband.
+- Add many more type annotations.
```

Comment:
There's no py.typed file so the annotations won't be seen

Reply:
That's intentional. Those annotations have been added for internal use only, mainly to verify unicode behaviour between functions. The "official" type annotations live in typeshed.

Reply:
Oh ok I think it's worth moving the types in here soon then

Reply:
Yup, thanks!